### DISCUSSION TITLE
[Question][Hw0]Cannot connect to the KCLOUD VM, error shows the connection times-out.

### DISCUSSION THREAD

**Discussion by yusungsim** [prosyslab-classroom/cs424-program-reasoning#375] [Q&A]
Hi.

I tried to connect the KCLOUD VM via ssh (the host "CS492H-20" in the sheet), but the server did not respond. The ssh command ends with the following error message. (erased out the IP address due to security reasons)

```sh
ssh: connect to host ***.***.***.*** port 22: Operation timed out
```

I tried to connect with and without the KVPN, but both tries failed in the same way. (btw, I'm using the KAIST internal network)

I guess there is a certain way to connect to the KCLOUD?

I hope students can successfully use the VM in short time.

**Comment by Hhro** [comment]
Hi.

To connect to KCLOUD, you should use "KCLOUD VPN" not "KVPN".

Here is a detailed guide: https://docs.google.com/presentation/d/1rS39M1dui3khqTc6joWniNrxH3x87nGgOrWHiMAL4DU 

Thanks.

**Comment by yusungsim** [comment]
It worked for me, thank you!
